script/install/debootstrap.bash

```diff
@@ -67,6 +67,11 @@ chroot "$ISOLATE_ROOT" apt-get install software-properties-common # provides add
     chroot "$ISOLATE_ROOT" add-apt-repository ppa:deadsnakes/ppa -y
   fi
 
+  if ! chroot "$ISOLATE_ROOT" apt-cache show pypy3 &>/dev/null; then
```

Comment:
We should unconditionally add this PPA, because even if a package named "pypy3" exists in standard repos it is probably old.